src/app/shell/personal-cabinet/shared-cabinet/messages/messages.component.ts

```diff
@@ -142,17 +144,17 @@ export class MessagesComponent extends CabinetDataComponent {
     });
   }
 
-  onEntitiesSelect(IDs: string[]): void {
-    this.chatRoomsParameters.workshopIds = IDs;
-    this.store.dispatch(new GetUserChatRooms(this.chatRoomsParameters));
+  onEntitiesSelect(workshopIds: string[]): void {
+    this.chatRoomsParameters.workshopIds = workshopIds;
```

Comment:
As I mentioned in the previous comment, it is better to add role as param, so we will have fewer arguments in the  function

I don't see any cons to why it should be separated. Also, you can assign this param on init